README.md

```diff
@@ -172,12 +172,14 @@ the command above will detect the os and architecture, dowload the archive, vali
 ./db1000n
 ```
 
-### k8s run
+### Kubernetes run
 
-Enter k8s directory
+## Simple run with deployment
+
+Enter to k8s simple deployment directory
```

Comment:
I'll merge it anyway but I think it should be "Enter k8s simple deployment directory", someone good at English should confirm that